### PR TITLE
[Pre-Training] Add some document for pre-training.

### PR DESCRIPTION
**Pull request by ZHUI**
### PR types

Others
### PR changes

Docs
### Description

[Pre-Training] Add some document for pre-training.

### CODE REVIEW

File: examples/language_model/ernie-1.0/README.md [hunk 1]
```diff
@@ -68,34 +68,38 @@ python -u  -m paddle.distributed.launch \
...
-ChnSentiCorp | 0.94495 | 0.95496 |
-PeoplesDailyNer | 0.95128 | 0.94035 |
-CMRC2018 | 72.05/85.67 | - |
+Model | Arch | CLUE 平均值 |  AFQMC | TNEWS | IFLYTEK | CMNLI | OCNLI | CLUEWSC2020 | CSL
+-- | -- | -- | -- | -- | -- | -- |  -- | -- | --
+Metrics |   |   |   | Acc | Acc | Acc | Acc | Acc | Acc | Acc
+ERNIE-1.0 Base | 12L768H | 73.78 |  74.95 | 58.73 | 61.37 | 81.77 | 75.46 | 81.25 | 82.93
+ERINE-1.0-cluecorpussmall | 12L768H | 73.24(-0.54) | 74.26 | 57.24 | 60.79 | 81.15 | 76.64 | 81.25 | 81.33
 
+注: `ERNIE-1.0 Base`采用的训练batch_size是1024，cluecorpussmall复现版本效果已经与base版本相差不大。
```
Comment (ZeyuChen) [inline examples/language_model/ernie-1.0/README.md:102]: 不要还用相差不大这种模糊的术语，不够严谨。

File: examples/language_model/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+| [BERT](./bert) | hdf5 | [链接](./pretraining_data_prepare)| 高  | 数据格式与官方Bert一致 |
+| [RoBERTa](./roberta) | txt转datasets cache | 无 | 中 | 数据处理转换为id后，cache保存，加载快 |
+| [ELECTRA](./electra) | 文本txt | 无 | 低 | 采用原始文本读取，每次load需要处理一遍数据，类似finetune流程 |
+| [BIG-BIRD](./bigbird) | 文本txt | 无 | 低 | 同上
+| [ConvBert](./convbert) | 文本txt | 无 | 低 | 同上
+
+
+其中，ERNIE-1.0 预训练功能支持较为完善。训练日志可视化，断点重启等功能支持较为完备。用户可以优先参考。
+
+## RoadMap
```
Comment (ZeyuChen) [inline examples/language_model/README.md:40]: Roadmap，这是个完整的单词

File: examples/language_model/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+# PaddleNLP 语言模型预训练
```
Comment (ZeyuChen) [inline examples/language_model/README.md:1]: 这份文档写的感觉比较像内部的技术文档，不适合作为开源对外。

**Comment by ZHUI** [comment]
Close https://github.com/PaddlePaddle/PaddleNLP/pull/2064, please refer https://github.com/PaddlePaddle/PaddleNLP/pull/2117

**Pull request closed without merging**